hazelcast-integration/spring-hibernate-2ndlevel-cache/pom.xml

```diff
@@ -28,35 +40,30 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
-			<exclusions>
-				<exclusion>
-					<groupId>org.junit.vintage</groupId>
-					<artifactId>junit-vintage-engine</artifactId>
-				</exclusion>
-			</exclusions>
 		</dependency>
 
 		<dependency>
 			<groupId>com.hazelcast</groupId>
 			<artifactId>hazelcast-hibernate53</artifactId>
-			<version>2.0.0</version>
+			<version>2.2.2-SNAPSHOT</version>
 		</dependency>
 
 		<dependency>
 			<groupId>com.hazelcast</groupId>
 			<artifactId>hazelcast</artifactId>
-			<version>4.0.1</version>
+			<version>5.3.0-SNAPSHOT</version>
```

Comment:
We can change this to `${hazelcast.version}` now.

Reply:
fixed: https://github.com/hazelcast/hazelcast-code-samples/pull/550/commits/2d112e9b61c8a3160c611160c8db79d0ab537ba3